requirements/extras/sqs.txt

```diff
@@ -1 +1,3 @@
-kombu[sqs]
+boto3>=1.9.12
+pycurl~=7.44.1
```

Comment:
Note that this version do not have any wheels. Latest version with wheels is [7.43.0.5](https://pypi.org/project/pycurl/7.43.0.5/#files)